.github/workflows/main.yml

```diff
@@ -308,7 +304,7 @@ jobs:
         run: cd "${{ github.workspace }}/build" && make check-unit
 
       - name: Run examples tests
-        run: cd "${{ github.workspace }}/build" && make check-examples
+        run: cd "${{ github.workspace }}/build" && make check-examples -j16
```

Comment:
shouldn't it be `-j$(nproc)` ?

Reply:
No, the examples tests are not CPU bound, they are I/O bound. For example, I use `-j99` on my 16 thread desktop and it's 4x faster than `-j17`. As long as it doesn't take more ram than the VM has, the larger the number is, the faster it goes.

Reply:
Maybe add a comment with this explanation?